src/test_application/DebugObject.cpp

```diff
@@ -111,22 +126,21 @@ void DebugCameraController::update_physics_pre()
     // When the camera is too far from the origin of the ActiveScene
     const int floatingOriginThreshold = 256;
 
-    Matrix4 &xform = m_scene.reg_get<active::ACompTransform>(m_ent).m_transform;
+    Matrix4 &xform = m_scene.reg_get<ACompTransform>(m_ent).m_transform;
 
     // round to nearest (floatingOriginThreshold)
     Vector3s tra(-xform.translation() / floatingOriginThreshold);
     tra *= floatingOriginThreshold;
 
     // convert to space int
-    tra *= gc_units_per_meter;
+    tra *= osp::gc_units_per_meter;
 
     if (!tra.isZero())
     {
         std::cout << "Floating origin translation!\n";
 
         // Move the active area to center on the camera
-        m_scene.dynamic_system_get<active::SysAreaAssociate>("AreaAssociate")
-                .area_move(-tra);
+        m_scene.dynamic_system_find<SysAreaAssociate>().area_move(-tra);
```

Comment:
Using std::negate would be more readable here: https://en.cppreference.com/w/cpp/utility/functional/negate

My eyesight sucks, almost missed that there was a minus sign there.